Docs: Add environment variable reference page (#472)
ROCm/rocSPARSE: docs/reference/env_variables.rst

.. meta::
  :description: rocSPARSE environment variables reference
  :keywords: rocSPARSE, ROCm, API, environment variables, environment, reference

.. _rocsparse_environment_variables_:

********************************************************************
rocSPARSE environment variables
********************************************************************

> In this section, the reader can find all the important rocSPARSE environment variables,

Reviewer: ```suggestion
This section describes the important rocSPARSE environment variables,
```

Author: Fixed

which are grouped by functionality.

Logging variables
================================================================================

The logging environment variables for rocSPARSE are collected in the following table.

> For more information, please check :ref:`rocsparse_logging`.

Reviewer: ```suggestion
For more information, see :ref:`rocsparse_logging`.
```

Author: Fixed
